src/briefcase/commands/base.py

```diff
@@ -673,6 +673,12 @@ def _add_update_options(
             help=f"Update requirements for the app{context_label}",
         )
 
+        parser.add_argument(
+            "--relock",
+            action="store_true",
+            help=f"Recalculate locked requirements for the app{context_label}",
+        )
```

Comment:
Why "relock" rather than simply "lock"? "Relock" implies that it was locked before, which isn't necessarily the case.

Reply:
In the way it's defined in the PR at present, it *is* a relock, because the initial create call always passes `relock=True`, automatically creating the lock file if the setting exists. This is part of my issue with this PR, as you need to specify a lock file that doesn't exist to get the initial invocation to work as you'd expect. So, "re" part of the relocking is present, but it won't always be clear how you get to the relock in the first place.

If the setting is also exposed to the `create` command, then it would definitely make sense for the option to be `--lock`. It might even make sense regardless, describing "locking" as an imperative action that is part of a build.